src/related-manager.ts

```diff
@@ -88,16 +89,30 @@ class RelatedManager extends KalturaPlayer.core.FakeEventTarget {
     this.plugin.player.play();
   }
 
-  playNext() {
+  playNext(seconds?: number) {
     this.logger.info('going to play next entry');
-    this.playByIndex(0);
+    this.clearNextEntryTimeout();
+
+    if (seconds && seconds > 0) {
+      this.nextEntryTimeoutId = window.setTimeout(() => {
+        this.playByIndex(0);
+      }, seconds * 1000);
+    } else {
+      this.playByIndex(0);
+    }
   }
 
   playSelected(internalIndex: number) {
     this.logger.info('going to play selected entry');
+    this.clearNextEntryTimeout();
```

Comment:
I think it's better to move it to `playByIndex` which could be called from other places

Reply:
didn't want to do it because then playNext would try to cancel it's own timeout inside the callback
which won't actually do anything but it's redundant

Reply:
I prefer to do it redundantly rather than do it in multiple places